_posts/2017-09-21-cockpit-151.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Cockpit 151
+author: pitti
+date: 2017-09-21 09:00
+tags: cockpit linux
+slug: cockpit-151
+category: release
+summary: Cockpit with Kubernetes fixes and SSH improvement
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 151.
+
+### Support loading SSH keys from arbitrary paths
+
+The User menu's Authentication dialog now supports entering arbitrary paths to
+SSH keys for adding to the SSH authentication agent. Previously this only
+offered keys present in the standard `~/.ssh` home directory.
+
+See it in action:
+
+<iframe width="960" height="720" src="https://cockpit.fedorapeople.org/add-keys.webm" frameborder="0" allowfullscreen></iframe>
+
+### Support X-Forwarded-Proto HTTP header for Kubernetes
+
+Newer Kubernetes versions support reading the
+[X-Forwarded-Proto](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Proto)
+HTTP header, which helps to determine whether or not a client used SSL to
+connect to an intermediate proxy, load balancer, or similar. Cockpit's
+Kubernetes (Cluster) dashboard now sets this header. Earlier versions have
+already done that for connecting to OpenShift.
```

Comment:
"already done that when hosted in Openshift"

Reply:
Updated, thanks!